web/api/v1/api.go

```diff
@@ -147,6 +148,8 @@ func (api *API) Register(r *route.Router) {
 
 	r.Get("/targets", instr("targets", api.targets))
 	r.Get("/alertmanagers", instr("alertmanagers", api.alertmanagers))
+
+	r.Get("/read", prometheus.InstrumentHandler("read", http.HandlerFunc(api.remoteRead)))
```

Comment:
I think it should be versioned somehow, yes.
Given that we eventually may want to move this to gRPC, the REST bindings for that would/should also go under the v2 API.